src/VerifyWindowsHelloController.js

```diff
@@ -95,7 +99,7 @@ function (Okta, FormController, FormType, webauthn, Spinner, FooterSignout, Wind
           'request': '_startEnrollment',
           'error': '_stopEnrollment',
           'abort': '_stopEnrollment',
-          'sync': '_successEnrollment'
```

Comment:
I think ```sync``` was used in line 53 to start the enrollment. But take a look at my other comment see if that makes sense.